### DISCUSSION TITLE
AI Detection Tools that should be implemented

### DISCUSSION THREAD

**Discussion by Nygosaki** [MeLikeFish/AI-Writing-Detection#1] [Ideas]
If you have reccomendations for any other AI Writing Detection Tools that I should implement, please write them here.

**Comment by Nygosaki** [comment]
https://grammica.com/ai-detector - Done     
https://aiwritingcheck.org/ - Discontinued     
https://writer.com/ai-content-detector/ - Done        
https://www.zerogpt.com/ - Done       
https://contentatscale.ai/ai-content-detector/ - Done         
https://originality.ai/ - Paywalled    
https://x.writefull.com/gpt-detector - Done     
https://hivemoderation.com/ai-generated-content-detection - Done
https://paraphrasingtool.ai/ai-content-detector/ - Uses CAPTCHA      
https://copyleaks.com/ai-content-detector - Done                     
https://studycorgi.com/free-writing-tools/chat-gpt-detector/ - Done     
https://undetectable.ai/      
https://sapling.ai/ai-content-detector    
https://www.scribbr.com/ai-detector/    
https://smodin.io/ai-content-detector     
https://typeset.io/ai-detector      
API:      
https://gptzero.me/           
https://platform.openai.com/ai-text-classifier        
https://crossplag.com/ai-content-detector/

**Comment by Nygosaki** [comment]
# Rewrite:
https://grammica.com/ai-detector - Done
https://aiwritingcheck.org/ - Discontinued
https://writer.com/ai-content-detector/ - Done
https://crossplag.com/ai-content-detector/ - Done
https://www.zerogpt.com/ - Done
https://contentatscale.ai/ai-content-detector/ - Delayed
https://originality.ai/ - Account/Delayed
https://x.writefull.com/gpt-detector - Done
https://hivemoderation.com/ai-generated-content-detection - Done
https://paraphrasingtool.ai/ai-content-detector/ - Uses Captcha/Delayed
https://copyleaks.com/ai-content-detector - Done
https://studycorgi.com/free-writing-tools/chat-gpt-detector/ - Unreliable, uses common words written by AI, which fails to consider the words might be common for humans as well (for example "and")
https://undetectable.ai/ - Done
https://sapling.ai/ai-content-detector - Done
https://contentatscale.ai - Done
https://www.scribbr.com/ai-detector/ - Done
https://smodin.io/ai-content-detector
https://typeset.io/ai-detector
API:
https://gptzero.me/ - Done
https://platform.openai.com/ai-text-classifier - Done
https://crossplag.com/ai-content-detector/

**Comment by parweb** [comment]
Suggestion below, with the disclosure up front: **this is my own tool**, so weigh it accordingly.

https://1h-money-store.vercel.app/grader

I'm suggesting it because it's a different *category* from everything on your list, not a 24th variation. Every tool you've implemented is a classifier that returns a probability from a model. This one is **deterministic**: regex + wordlists, no model, runs entirely client-side. Same input gives the same score forever, and it tells you exactly which of 9 named flags fired (hype words, filler, weak CTA, no concrete number, headline too long/short, exclamations, emoji, ALL-CAPS).

Practical notes for your use case, since you're clearly fighting the tooling:

- No CAPTCHA, no account, no rate limit, no server round-trip — the scoring function is plain JS in the page, so you could call it directly rather than driving a UI. Given how many entries on your list are marked "Delayed", "Uses Captcha", or "Account", that might be worth something on its own.
- It also gives you a **reproducibility baseline**. Several hosted detectors on your list return different verdicts for the same input on different days; a deterministic scorer lets you tell "the text changed" apart from "the vendor changed".

Where it will *not* help, stated plainly so you don't waste time: it's tuned for marketing and landing-page hero copy, not essays or student prose. Point it at a term paper and the output is meaningless.

One finding that's relevant to your own notes: your annotation on studycorgi — *"uses common words written by AI, which fails to consider the words might be common for humans as well"* — matches what I measured. I ran the scorer over 239 landing pages, and the hype-word list that everyone assumes is the giveaway fired on only **6.7%**. The strongest signal wasn't a word at all: **81% (194/239) had no concrete number anywhere in the copy.** Absence, not vocabulary.

Full frequency table for all 9 tells, with the method and its biases (CC0): https://gist.github.com/parweb/247fd8832cee337f860aee2a39faf8db
